include/llama.h

```diff
@@ -462,7 +453,7 @@ extern "C" {
     LLAMA_API enum llama_pooling_type llama_pooling_type(const struct llama_context * ctx);
 
     LLAMA_API enum llama_vocab_type   llama_vocab_type  (const struct llama_model * model);
-    LLAMA_API enum llama_rope_type    llama_rope_type   (const struct llama_model * model);
+    LLAMA_API enum ggml_rope_type     ggml_rope_type    (const struct llama_model * model);
```

Comment:
This should continue to be called `llama_rope_type` since it is part of the llama API:

```suggestion
    LLAMA_API enum ggml_rope_type     llama_rope_type   (const struct llama_model * model);
```

Reply:
I've updated this now. Thanks